packages/volto/src/helpers/Blocks/Blocks.js

```diff
@@ -629,51 +629,85 @@ export const styleDataToStyleObject = (key, value, prefix = '') => {
  * Generate styles object from data
  *
  * @function buildStyleObjectFromData
- * @param {Object} obj A style wrapper object data
+ * @param {Object} data A block data object
  * @param {string} prefix The prefix (could be dragged from a recursive call, initially empty)
  * @return {Object} The style object ready to be passed as prop
  */
-export const buildStyleObjectFromData = (obj = {}, prefix = '') => {
+export const buildStyleObjectFromData = (data = {}, prefix = '') => {
```

Comment:
You could easily make it non-breaking by making it accept either `obj` or `data`

Reply:
Done, I had to detect that we have a full block by testing if `@type` is present.